nvidia_entrypoint.sh

```diff
@@ -81,8 +81,13 @@ fi
 
 echo
 
-if [[ $# -eq 0 ]]; then
-  exec "/bin/bash"
+# If detect Vertex AI environment, launch tritonserver with supplied arguments
+if [[ ! -z "$AIP_MODE" ]]; then
+  exec /opt/tritonserver/bin/tritonserver "$@"
```

Comment:
Strange that you need to full path, because "docker run <image> tritonserver --model-repository=<x>" works.
I'm concerned that if full path is required then does that mean that the env is not set property as well?

Reply:
It may not need the full path, I changed it to resolve the "not found" issue but it later seems to be related to exec usage and the the way quote ("") should be used (quote on the whole thing will make linux use the full string for executable name). I didn't revert the change just in case.